app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -92,12 +115,44 @@ class ResponseRedacter
           fraudpoint.synthetic_identity_index
           fraudpoint.transaction_status
           fraudpoint.vulnerable_victim_index
+          fuzzy_device_first_seen
+          fuzzy_device_id_confidence
+          fuzzy_device_last_event
+          fuzzy_device_last_update
+          fuzzy_device_match_result
+          fuzzy_device_result
+          fuzzy_device_score
+          fuzzy_device_worst_score
+          http_connection_type
+          http_referer
```

Comment:
I think we should avoid logging referer if we can? It's not explicitly PII but kind of could be, some times?

Reply:
It *should* only be our own pages right?

Reply:
The `http_referer` field I'm getting back from LN appears to contain an encoded or encrypted value, which is not what I was expecting. I'm removing it because I'm uncertain what it represents.